src/test/java/duke/commands/DateCommandTest.java

```diff
@@ -0,0 +1,24 @@
+package duke.commands;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.tasks.TaskList;
+
+
+public class DateCommandTest {
+    private Command command = new DateCommand("2020-09-20");
+    private Storage storage = new Storage();
+    private TaskList list = new TaskList();
+
+    @Test
+    public void dateDoesNotExist_exceptionThrown() {
```

Comment:
The `featureUnderTest_testScenario_expectedBehavior()` format should, at least, include the 1st section of the feature under test. Perhaps `dateCommandExecution_dateDoesNotExist_exceptionThrown` might be a better way to name this?